### PR TITLE
Update node local dns cache

### PR DESCRIPTION
**Pull request by zetaab**

### CODE REVIEW

File: upup/models/cloudup/resources/addons/nodelocaldns.addons.k8s.io/k8s-1.12.yaml.template [hunk 1]
```diff
@@ -192,3 +192,24 @@ spec:
           items:
             - key: Corefile
               path: Corefile.base
+---
```
Comment (olemarkus) [inline upup/models/cloudup/resources/addons/nodelocaldns.addons.k8s.io/k8s-1.12.yaml.template:195]: We don't ship these headless services typically used by prom operator as we don't ship prom operator either.

Also, `PodMonitor` avoids the need for these services.

**Review by olemarkus — CHANGES_REQUESTED** [review]
Looks good except for the headless service that I don't feel belongs as part of the addon.

**Comment by zetaab** [comment]
@olemarkus I just updated the node local dns, maybe that is enough for now.

**Review by olemarkus — APPROVED** [review]
Thanks man

**Pull request merged**